Project.toml

```diff
@@ -3,11 +3,18 @@ uuid = "b3c78db8-5351-4c94-92cb-e6a7a02f3abe"
 authors = ["JieLi"]
 version = "0.1.0"
 
+[deps]
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+NiLang = "ab4ef3a6-0b42-11ea-31f6-e34652774712"
```

Comment:
Try moving NiLang to extras.

Reply:
For this I've opened a new issue #9, I think this can be fixed separately without blocking this PR.

If this can be fixed very quickly then by all means do it here.

Reply:
Both sounds good.